apps/wallet/src/ui/app/pages/home/nft-details/index.tsx

```diff
@@ -128,39 +83,36 @@ function NFTDetailsPage() {
                                     <div className="flex flex-col gap-md">
                                         <div className="flex flex-col gap-xxxs">
                                             <span className="text-title-lg text-neutral-10 dark:text-neutral-92">
-                                                {nftDisplayData?.name}
+                                                {nftMeta?.name}
```

Comment:
More than replacing everywhere `nftDisplayData` with `nftMeta` is more accurate keeping the old name `nftDisplayData` because it is actually not just the metadata, this is about the display field **only**. If you look at https://github.com/iotaledger/iota/blob/78517eefd2f36a837bb6d831677f5b3d437c40c1/apps/core/src/hooks/useGetNFTMeta.ts#L14 you can see how it actually is just the display standard, not a generic metadata. So I would either keep it as it was  before, and if you want we can rename `useGetNFTMeta` to `useGetNFTDisplay` and improve namings inside that hook